config/prod/neurolincs-actual.yaml

```diff
@@ -0,0 +1,22 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v4.yaml
```

Comment:
Use new template

Reply:
Is v4 no longer supported?

Reply:
I don't think its not supported, but you always want to use the newer versions as they become available.

Reply:
I understand, but v4 works for my needs. If you no longer want people to open PRs using the v4 or v3 templates you should deprecate and remove them from the templates directory. I'll be happy to reopen this PR using v5 if you decide to do so.

Reply:
both of you are correct.  I forgot to deprecate managed-ec2-v4.yaml.  It is deprecated now. https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/templates/managed-ec2-v4.yaml